### DISCUSSION TITLE
Can we close the cephx with rook?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#9743] [Q&A] [answered]
@travisn

**Comment by leseb** [comment, marked as answer]
Hopefully, I got the question right given its poor description. If you are looking at disabling cephx, you can follow this https://rook.io/docs/rook/v1.8/ceph-advanced-configuration.html#custom-cephconf-settings and apply this https://docs.ceph.com/en/latest/rados/configuration/auth-config-ref/#disabling-cephx in your ceph.conf.

Thanks.

**Comment by zhucan** [comment]
@leseb Thanks，I will take a look.

**Comment by zhucan** [comment]
Only can edit the configmap to modify the ceph.conf? It's psssible when creating a configmap, customize ceph.conf, because we want to customize ceph.conf before we create the osd or pool. @leseb

**Reply by leseb** [reply]
Then yes create the ConfigMap before deploying the OSDs.

**Reply by zhucan** [reply]
Is there a doc for this?

**Reply by leseb** [reply]
https://rook.io/docs/rook/v1.8/ceph-advanced-configuration.html#custom-cephconf-settings
